purchase_order_product_recommendation/wizards/purchase_order_recommendation.py

```diff
@@ -152,6 +152,13 @@ def _prepare_wizard_line(self, vals, order_line=False):
                                qty_to_order),
             'is_modified': vals.get('is_modified', False),
         }
+        vals['price_unit'] = product_id._select_seller(
```

Comment:
This is only syntactic sugar, but you can still return the dictionary, but adding the key with the call to `_select_seller`.

Reply:
Yes, it can be made in a prettier way :) Change